validation/deleate_only_create_resources/delte_only_create_resources.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+	"time"
+
+	tap "github.com/mndrix/tap-go"
+	"github.com/mrunalp/fileutils"
+	rspec "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/opencontainers/runtime-tools/specerror"
+	"github.com/opencontainers/runtime-tools/validation/util"
+	uuid "github.com/satori/go.uuid"
+)
+
+func main() {
+	t := tap.New()
+	t.Header(0)
+
+	// Create a cgroup
+	cgPath := "/sys/fs/cgroup"
+	pidsPath := filepath.Join(cgPath, "pids")
+	os.Mkdir(filepath.Join(pidsPath, "cgrouptest"), 0755)
+
+	bundleDir, err := util.PrepareBundle()
+	if err != nil {
+		util.Fatal(err)
+	}
+	defer os.RemoveAll(bundleDir)
+
+	r, err := util.NewRuntime(util.RuntimeCommand, bundleDir)
+	if err != nil {
+		util.Fatal(err)
+	}
+
+	r.SetID(uuid.NewV4().String())
+	g, err := util.GetDefaultGenerator()
+	if err != nil {
+		util.Fatal(err)
+	}
+
+	err = r.SetConfig(g)
+	if err != nil {
+		util.Fatal(err)
+	}
+	err = fileutils.CopyFile("runtimetest", filepath.Join(r.BundleDir, "runtimetest"))
+	if err != nil {
+		util.Fatal(err)
+	}
+
+	err = r.Create()
+	if err != nil {
+		util.Fatal(err)
+	}
+
+	state, err := r.State()
+	if err != nil {
+		util.Fatal(err)
+	}
+	// Add the container to the cgroup
+	ioutil.WriteFile(filepath.Join(pidsPath, "cgrouptset/tasks"), []byte(strconv.Itoa(state.Pid)), 0644)
```

Comment:
typo: "cgrouptset" != "cgrouptest" from line 26. You could check the return value of `ioutil.WriteFile` and use a constant for that directory.